src/main/java/frc/robot/subsystems/PizzaBoxSubsystem/PizzaBoxSubsystem.java

```diff
@@ -19,17 +20,53 @@ public PizzaBoxSubsystem() {
 
   }
 
-  public Command reset_command(Servo s, Servo s2, TalonFX f) {
-    return this.runOnce(() -> PizzaBoxFuncs.reset(PBservo,PBFlapServo,m_PizzaBoxMotor));
+  public Command reset_command() {
+    return this.runOnce(() -> reset(PBservo,PBFlapServo,m_PizzaBoxMotor));
+  }
+
+  public Command spit_command(TalonFX motor) {
```

Comment:
PizzaBoxSubsystem commands should not need a parameter to take in a motor. You can access the m_PizzaBoxMotor here directly

Reply:
public Command spit_command() {
  return this.run(() -> m_PizzaBoxMotor.set(1));
}